src/tv2-common/migrations/index.ts

```diff
@@ -173,6 +174,74 @@ export function mapGfxTemplateToDesignTemplateAndDeleteOriginals(
 	})
 }
 
+export function moveSelectedGfxSetupNameToGfxDefaults(versionStr: string, fromValue: string, targetTable: string) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.moveSelectedGfxSetupName.${fromValue}.ToTable.${targetTable}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const singleValue = (context.getBaseConfig(fromValue) as unknown) as string
+			const designatedTable = (context.getBaseConfig(targetTable) as unknown) as TableConfigItemValue | undefined
+
+			if (!singleValue || !Array.isArray(designatedTable)) {
+				return false
+			}
+
+			return singleValue
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const singleValue = context.getBaseConfig(fromValue) as BasicConfigItemValue
+			const designatedTable = (context.getBaseConfig(targetTable) as unknown) as TableConfigItemValue
+			const setupName = 'DefaultSetupName'
+
+			designatedTable[0][setupName] = singleValue
+
+			context.setBaseConfig(targetTable, (designatedTable as unknown) as ConfigItemValue)
+			context.removeBaseConfig(fromValue)
+		}
+	})
+}
+
+export function moveSelectedGfxSetupNameToGfxDefaultsInVariants(
+	versionStr: string,
+	fromValue: string,
+	targetTable: string
+) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.moveSelectedGfxSetupNameToGfxDefaultsInVariants.${fromValue}.ToTable.${targetTable}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const allVariants = context.getAllVariants()
+			const checkForDefaultsTransfer = allVariants.some((variant: IBlueprintShowStyleVariant) => {
+				const defaultsTable = context.getVariantConfig(variant._id, targetTable) as TableConfigItemValue
+				const setupNameTable = context.getVariantConfig(variant._id, fromValue) as ConfigItemValue
+				return setupNameTable && !defaultsTable
+			})
+
+			return checkForDefaultsTransfer
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const allVariants = context.getAllVariants()
+			allVariants.forEach((variant: IBlueprintShowStyleVariant) => {
+				const singleValue = context.getVariantConfig(variant._id, fromValue) as BasicConfigItemValue
+				let designatedTable = context.getVariantConfig(variant._id, targetTable) as TableConfigItemValue
+				const setupName = 'DefaultSetupName'
+
+				if (designatedTable === undefined) {
+					designatedTable = [{ ['']: singleValue, _id: '' }]
+				}
+
+				designatedTable[0][setupName] = singleValue
+				const updatedVariant = variant
+
+				updatedVariant.blueprintConfig[targetTable] = designatedTable
+				context.updateVariant(variant._id, updatedVariant)
```

Comment:
I think you could just do something like 
```suggestion
				context.setVariantConfig(variant._id, targetTable, designatedTable)
```

Reply:
Changed it!